### DISCUSSION TITLE
test_console_logging for request error

### DISCUSSION THREAD

**Discussion by alicesin615** [seleniumbase/SeleniumBase#3112] [Q&A] [answered]
Hi, I saw the example `test_console_logging`  and I was wondering if there could be a way to capture the response from twitter when sb tries to open a twitter url by using the `start_recording_console_logs()` and `get_recorded_console_logs()` ? aim is to detect the response error status code from twitter before running the rest of my service to handle twitter's rate limit.

**Comment by mdmintz** [comment, marked as answer]
The page must finish loading before `self.start_recording_console_logs()` can be called.
Then call `self.get_recorded_console_logs()` before leaving the page.
